cmd/serve.go

```diff
@@ -76,7 +76,7 @@ var serveCmd = &cobra.Command{
 		case "consul":
 			db = consul.New(viper.GetString("jobdb-address"))
 		case "postgres":
-			dsn := fmt.Sprintf("postgres://%s:%s@%s", viper.GetString("jobdb-username"), viper.GetString("jobdb-password"), viper.GetString("jobdb-address"))
+			dsn := fmt.Sprintf("postgres://%s:%s@%s?sslmode=%s", viper.GetString("jobdb-username"), viper.GetString("jobdb-password"), viper.GetString("jobdb-address"), viper.GetString("sslmode"))
```

Comment:
Can't the person just do `--jobdb-address=<address>?sslmode=false`?  I don't think it's the best idea to start including a bunch of flags when query params can just be passed straight in using an existing flag.